src/rpc/rawtransaction.cpp

```diff
@@ -111,26 +112,27 @@ static RPCHelpMan getrawtransaction()
 {
     return RPCHelpMan{
                 "getrawtransaction",
-                "\nReturn the raw transaction data.\n"
+                "\nReturn the raw transaction data.\n\n"
```

Comment:
```suggestion
                "Return the raw transaction data.\n\n"
```

nit: no need for this `\n` after rebase